h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -236,17 +244,17 @@ abstract protected class Driver extends H2O.H2OCountedCompleter<Driver> {
 
     protected Driver(){ super(); }
     
-    private ModelBuilderListener _callback;
-
-    public void setCallback(ModelBuilderListener callback) {
-      this._callback = callback;
```

Comment:
callbacks have been moved from Driver to ModelBuilder itself as it allows much simple configurability needed by the pipeline